apps/web/pages/settings/my-account/appearance.tsx

```diff
@@ -74,8 +77,12 @@ const AppearanceView = () => {
       await utils.viewer.me.invalidate();
       showToast(t("settings_updated_successfully"), "success");
     },
-    onError: () => {
-      showToast(t("error_updating_settings"), "error");
+    onError: (error) => {
+      if (error.message) {
+        showToast(error.message, "error");
```

Comment:
Added the ability to show a custom error message. Needed here because the messages for the booker layout are quite helpful.